### PR TITLE
Harmonize PET tracers handling (ML / DL / Stats)

### PR DESCRIPTION
**Pull request by alexandreroutier**
Follows PR #129.

After this PR, handling of PET tracers will be the same across Clinica pipelines (`acq_label` / `suvr_reference_region` / optional `use_pvc_data`).

### CODE REVIEW

**Comment by alexandreroutier** [comment]
Hi @mdiazmel,

This PR is now ready for technical review. :)

I will finish ASAP the documentation so that @nburgos can do the Wiki review.

Alex

**Review by nburgos — COMMENTED** [review]
I checked all the wiki pages and made modifications when needed. Should be all good now.

File: clinica/utils/inputs.py [hunk 1]
```diff
@@ -447,13 +447,29 @@ def fetch_file(remote, dirname=None):
...
+    Download file from server
+
+    Args:
+        remote_file (str): RemoteFileStructure defined in clinica.utils.inputs
+        cache_path (str): (default: ~/.cache/clinica/data)
+
+    Returns:
+        Path to downloaded file.
+
+    Note:
```
Comment (mdiazmel) [inline clinica/utils/inputs.py:461]: To remove!

File: clinica/utils/inputs.py [hunk 2]
```diff
@@ -447,13 +447,29 @@ def fetch_file(remote, dirname=None):
...
     from clinica.utils.stream import cprint
 
     home = str(Path.home())
-    cache_clinica = os.path.join(home, ".cache", "clinica", "data")
+    if cache_path:
+        cache_clinica = os.path.join(home, ".cache", cache_path)
+    else:
+        cache_clinica = os.path.join(home, ".cache", "clinica", "data")
     if not (os.path.exists(cache_clinica)):
         os.makedirs(cache_clinica)
```
Comment (mdiazmel) [inline clinica/utils/inputs.py:474]: Add the trick of `exist_ok=True` to avoid the `if` instruction.

**Review by mdiazmel — COMMENTED** [review]
Thanks @alexandreroutier,
Code readability has improved a lot with these changes!
A couple suggestions before merging.

**Comment by alexandreroutier** [comment]
> Thanks @alexandreroutier,
> Code readability has improved a lot with these changes!
> A couple suggestions before merging.

Hi!

Thanks for the feedback, I addressed the comments in 022b72e :)

I think this can now be merged.

**Comment by mdiazmel** [comment]
Hi @alexandreroutier ,
I propose to do a "squash and merge" for this PR, if you're agree.

**Pull request merged**